class-wc-calypso-bridge.php

```diff
@@ -96,6 +97,29 @@ public function deactivate_duplicate_tiktok() {
 		}
 	}
 
+	/**
+	 * Deactivate WooCommerce Services if either Woo Shipping or Woo Tax is active on an ecommerce-related plan
+	 * (on WPCOM and Woo Express, including trial plans).
```

Comment:
[Not blocking] Assuming we default to WP code standards, then the summary should be a single line with a new line separating to the description.

I've made some a suggestion that tries to separate it that way, but feel free to ignore any of my wording 😄 

If we have any P2 post documenting this logic, then it would be an awesome `@link` DocBlock tag to include as well!

```suggestion
	 * Deactivate WCS&T if Woo Shipping or Woo Tax is active on an atomic  ecommerce site.
	 *
 	 * This logic is meant to be applied to all ecommerce sites on Atmoic (read: it's not exclusive to Woo Express).
```

Reply:
Thanks! I've added a link to this PR and in the PR description, I added a Slack link with the reasoning.